Makefile

```diff
@@ -169,7 +169,7 @@ test-with-database:
 .PHONY: test-unit-and-integration
 test-unit-and-integration:
 	export GLOBIGNORE="$(GLOBIGNORE)";\
-	timeout -v ${TIMEOUT_RETRIES} tools/retry prove ${PROVE_LIB_ARGS} ${PROVE_ARGS}
+	timeout -s SIGINT -k 5 -v ${TIMEOUT_RETRIES} tools/retry prove ${PROVE_LIB_ARGS} ${PROVE_ARGS}
```

Comment:
Is the `-k 5`actually needed here?

Reply:
I guess it makes sense If the process is really stuck.

Reply:
fine but then it is a different change, right?

@kalikiana I don't mind the kill option but please make that explicit, e.g. a separate commit. And also please write `prove` in lowercase :)